bindings/python/Makefile.am

```diff
@@ -31,8 +31,9 @@ DISTCLEANFILES =                                   \
    $(top_builddir)/bindings/python/__init__.py
 
 if HAVE_SWIG
-$(BUILT_SOURCES) $(pkgpython_PYTHON): $(SWIG_SOURCES)
-	$(SWIG) -python -module clinkgrammar -I$(top_srcdir)/link-grammar -o $@ $<
+#$(BUILT_SOURCES) $(pkgpython_PYTHON): $(SWIG_SOURCES)
+$(BUILT_SOURCES) : $(SWIG_SOURCES)
+	$(SWIG) -python -py3 -module clinkgrammar -I$(top_srcdir)/link-grammar -o $@ $<
```

Comment:
It would be better make a separate project that could be reached from Python Package Index, IMO.